src/gp_init.c

```diff
@@ -376,47 +376,101 @@ int init_event_fini(struct gssproxy_ctx *gpctx)
     return 0;
 }
 
-void init_proc_nfsd(struct gp_config *cfg)
+static int init_proc_nfsd_once(void)
```

Comment:
Why call it _once() if it gets called multiple times? :)

Reply:
As in "make one attempt"....I'll rename it....`try_init_proc_nfsd` maybe?